generators/utils.js

```diff
@@ -472,9 +486,63 @@ function checkStringInFile(path, search, generator) {
     return fileContent.includes(search);
 }
 
+/**
+ * Loads the blueprint information from the configuration of the specified generator.
+ * @param config - the generator's configuration object.
+ * @returns {Array} an array that contains the info for each blueprint
+ */
+function loadBlueprintsFromConfiguration(generator) {
+    // load blueprints from config file
+    const blueprints = generator.config.get('blueprints') || [];
+
+    const oldBlueprintName = generator.config.get('blueprint');
+    if (oldBlueprintName) {
+        const version = generator.config.get('blueprintVersion') || 'latest';
+        blueprints.push(parseBlueprintInfo(`${oldBlueprintName}@${version}`));
```

Comment:
Maybe do this only if `oldBlueprintName` is not also declared in `blueprints` sections of config file?

Reply:
Fixed